### DISCUSSION TITLE
config.yaml templates displayed in Dashbord Text Adaptation window do not have newlines

### DISCUSSION THREAD

**Discussion by sjpritchard** [speechly/speechly#311] [Q&A] [answered]
I'm using the config.yaml file. When I deploy my config, the templates from my config file are rendered as a single concatenated line in the Text Adaption field of the Dashboard, eg:

![image](https://github.com/speechly/speechly/assets/1045715/d3c8283e-e16c-48b6-83a8-edd874627fc6)

**Comment by bigdatabaracus** [comment]
Thanks @sjpritchard for letting us know. @teelisyys would you have an idea what's going on?

**Reply by teelisyys** [reply, marked as answer]
Hi @sjpritchard !

Can you please verify that you are using the _literal style_ block scalar for the `templates` block in the `config.yaml`? Like this (note `|` after `templates:`):

~~~
templates: |
  *turn_on [turn | switch] on the $device_as_spoken(device)
  *turn_on [turn | switch] the $device_as_spoken(device) on
~~~

If you are using the _folded style_ block scalar in yaml (i.e. `>` after `templates:`), single newlines are replaced with spaces and you must use empty line as a separator for the examples, like so:

~~~
templates: >
  *turn_on [turn | switch] on the $device_as_spoken(device)

  *turn_on [turn | switch] the $device_as_spoken(device) on
~~~

**Reply by sjpritchard** [reply]
I'm using neither of those, and am using the style in your documentation which is '>-'

Your docs:
![image](https://github.com/speechly/speechly/assets/1045715/4e2d8e8a-504d-498f-a07e-b46000326f35)

**Reply by sjpritchard** [reply]
I'll try with the pipe symbol instead.

**Reply by sjpritchard** [reply]
Confirming that replacing it with | fixes it.

**Reply by teelisyys** [reply]
Excellent! The documentation has now been fixed as well.
